components/authz-service/server/project_update_manager.go

```diff
@@ -48,6 +48,7 @@ var SerializedProjectUpdateDomainServices = []string{
 	"compliance",
 	"ingest",
 	"nodemanager",
+	"feed",
```

Comment:
Adds the event feed to one of the services that are called from authz during a project update.